src/builder.rs

```diff
@@ -917,6 +917,18 @@ pub struct SigningParts {
     alpha: pallas::Scalar,
 }
 
+#[cfg(feature = "unstable-frost")]
+impl SigningParts {
+    /// Return the spend validating key for this action.
+    pub fn ak(&self) -> &SpendValidatingKey {
+        &self.ak
+    }
+    /// Return the randomization for this action.
+    pub fn alpha(&self) -> pallas::Scalar {
```

Comment:
These getters are only necessary for the older compile-time state machine builder API. Since this PR was opened, we've deployed PCZTs which already include these getters implicitly, and PCZTs (and runtime state machine builder APIs in general) are the general direction we will be heading, so these aren't technically necessary. That being said, as these are behind an unstable feature flag I wouldn't mind them going in for testing purposes, on the understanding that they likely will eventually be removed (along with the corresponding builder logic).